src/main/resources/rs117/hd/scene/model_overrides.json

```diff
@@ -2759,7 +2759,7 @@
   },
   {
     "description": "Wintertodt Brazier",
-    "baseMaterial": "METALLIC_2_HIGHGLOSS",
+    "baseMaterial": "_HIGHGLOSS",
```

Comment:
Slight mistake here. Was this meant to be `METALLIC_2_HIGHGLOSS`?

Reply:
Oh good catch! 
It was meant to be metallic_1_light_highgloss

Metallic_2 was just metallic_1 lightened so I renamed it.